website/content/en/docs/setup/quickstart.md

```diff
@@ -165,15 +209,27 @@ We're just scratching the surface in this post. To get your hands dirty with Vec
 * Finally, [deploying Vector][deployment] to launch Vector in your production environment.
 
 [components]: /components
+
```

Comment:
Did you mean to add all of these extra blank lines?

Reply:
Reverted all the formatting changes.